### DISCUSSION TITLE
1.0.0 released with new module URL: dario.cat/mergo

### DISCUSSION THREAD

**Discussion by darccio** [darccio/mergo#239] [Announcements]
This PR is a release containing [0.3.15](https://github.com/imdario/mergo/releases/tag/v0.3.15) features but changing the module URL to `dario.cat/mergo`.

## What's Changed
* feat: v1.0.0 released by @imdario in https://github.com/imdario/mergo/pull/238

**Full Changelog**: https://github.com/imdario/mergo/compare/v0.3.16...v1.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/imdario/mergo/releases/tag/v1.0.0'>1.0.0 released with new module URL: dario.cat/mergo</a>.</em>

**Comment by phuslu** [comment]
user comments were deleted?

**Reply by darccio** [reply]
@phuslu Comments were in a separate thread https://github.com/darccio/mergo/discussions/236#discussioncomment-7009509
